app/main.py

```diff
@@ -1,2 +1,60 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(self,
+                 capacity: int = 8,
+                 size: int = 0,
+                 load_factor: float = 2 / 3
+                 ) -> None:
+        self.capacity = capacity
+        self.size = size
+        self.load_factor = load_factor
+        self.storage = [(None, None, None)] * self.capacity
```

Comment:
There should be no need to store tuple of three `None` as a default state, only `None` instead of tuple should be sufficient. Make according change here and in the `resize` method as well